src/dbally/data_models/llm_options.py

```diff
@@ -9,13 +11,29 @@ class LLMOptions:
     described in the [OpenAI API documentation](https://platform.openai.com/docs/api-reference/chat/create.)
     """
 
-    frequency_penalty: Optional[float]
-    max_tokens: Optional[int]
-    n: Optional[int]
-    presence_penalty: Optional[float]
-    seed: Optional[int]
-    stop: Optional[Union[str, List[str]]]
-    temperature: Optional[float]
-    top_p: Optional[float]
+    frequency_penalty: Union[Optional[float], NotGiven] = NOT_GIVEN
+    max_tokens: Union[Optional[int], NotGiven] = NOT_GIVEN
+    n: Union[Optional[int], NotGiven] = NOT_GIVEN
+    presence_penalty: Union[Optional[float], NotGiven] = NOT_GIVEN
+    seed: Union[Optional[int], NotGiven] = NOT_GIVEN
+    stop: Union[Optional[Union[str, List[str]]], NotGiven] = NOT_GIVEN
+    temperature: Union[Optional[float], NotGiven] = NOT_GIVEN
+    top_p: Union[Optional[float], NotGiven] = NOT_GIVEN
 
     dict = asdict
+
+    def __or__(self, other: "LLMOptions") -> "LLMOptions":
```

Comment:
Can we have a test for it? Looks good but if there is an issue with this, finding such a bug can be not so funny

Reply:
The merge functionality has been removed.